File: src/DatingLoveApp.Business/Dtos/LocalUserDtos/LocalUserDto.cs
﻿namespace DatingLoveApp.Business.Dtos.LocalUserDtos;

public class LocalUserDto
{
    public Guid LocalUserId { get; set; }

    public string UserName { get; set; } = null!;

    public DateTime DateOfBirth { get; set; }

    public string Gender { get; set; } = null!;

    public string Nickname { get; set; } = null!;

    public string? ProfilePictureUrl { get; set; }

    public string? Address { get; set; }

    public string? Ward { get; set; }

    public string? District { get; set; }

    public string? City { get; set; }
}

File: client/src/app/_dtos/localUserDtos/localUserDto.ts
export interface LocalUserDto {
    localUserId: string;
    userName: string;
    dateOfBirth: string;
    gender: string;
    nickname: string;
    profilePictureUrl: string | null;
    address: string | null;
    ward: string | null;
    district: string | null;
    city: string | null;
}